circle.yml

```diff
@@ -29,13 +29,16 @@ deployment:
   prequel:
     branch: development
     commands:
+      - docker build -f docker/reaction.prod.docker -t reactioncommerce/prequel:$CIRCLE_BUILD_NUM .
+      - docker tag reactioncommerce/prequel:$CIRCLE_BUILD_NUM reactioncommerce/prequel:latest
       - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - docker tag reactioncommerce/prequel reactioncommerce/prequel:$CIRCLE_BUILD_NUM
       - docker push reactioncommerce/prequel:$CIRCLE_BUILD_NUM
+      - docker push reactioncommerce/prequel:latest
   release:
     branch: master
     commands:
-      - docker build -t reactioncommerce/prequel .
+      - docker build -f docker/reaction.prod.docker -t reactioncommerce/reaction:$CIRCLE_BUILD_NUM .
+      - docker tag reactioncommerce/reaction:$CIRCLE_BUILD_NUM reactioncommerce/reaction:latest
       - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - docker tag reactioncommerce/prequel reactioncommerce/reaction:$CIRCLE_BUILD_NUM
```

Comment:
Because the CI build that happens first is not using the production Dockerfile.  It uses a slightly different CI Dockerfile.  The exact same bash scripts are used in both cases, but the CI build caches all of the OS dependencies in their own layer so that subsequent builds are much faster.  As a result, the cleanup script isn't as efficient in the CI image and the container ends up being ~170MB larger.    

Also, that second build with the production Dockerfile only happens for code pushed to development or master branches.